python/rikai/types/geometry.py

```diff
@@ -115,6 +116,12 @@ def __init__(self, xmin: float, ymin: float, xmax: float, ymax: float):
         self.ymin = float(ymin)
         self.xmax = float(xmax)
         self.ymax = float(ymax)
+        dict.__init__(self, {
```

Comment:
does this store the values 2 times?

Reply:
Wondering whether a box is both an array and dict breaks sth?
like `box[0] == box["xmin"] == box.xmin` .

Reply:
Oh for Box2d this was some leftover hack. No need for Box2d to subclass both.